src/schema.py

```diff
@@ -127,3 +127,5 @@ class BackendStatusDatasetsSchema(BaseModel):
 
 class ServerSchema(BaseModel):
     version: str
+    openid_auth_url: str
```

Comment:
Great call! This is actually a bug further down in typing (`HGET` can return None), and would crash some instances if someone updated their instance carelessly (because this request is now done from every subpage).

Of course I plan to test everything before the release but still, nice to have a working master branch. I'll add tests to ensure things like this won't happen again.

Reply:
(fixed in the new versioN0